Source/ModelGroups/Material_Mixed.cs

```diff
@@ -1,236 +1,103 @@
+        public Material_Mixed(List<string> imageList)
         {
-            // Removes the empty set models. Don't need the automatic one for this set.
-            combos.RemoveAt(0);
+            var baseColorTextureImage = GetImage(imageList, "BaseColor_X");
+            GetImage(imageList, "UVSpace2", isFigure: true);
+            GetImage(imageList, "UVSpace3", isFigure: true);
 
-            // Sort the combos by complexity
-            combos.Sort(delegate (List<Property> x, List<Property> y)
-            {
-                if (x[0].value == null &&
-                    x[1].value == null) return -1; // Spec gloss on all
-                else if (x[0].value == null &&
-                         x[1].value != null) return 1; // Split application
-                else return 0;
-            });
+            // Track the common properties for use in the readme.
+            CommonProperties.Add(new Property(PropertyName.ExtensionUsed, "Specular Glossiness"));
+            CommonProperties.Add(new Property(PropertyName.BaseColorTexture, baseColorTextureImage));
 
-            return combos;
-        }
+            Model CreateModel(Action<List<Property>, Runtime.Material, Runtime.Material> setProperties)
+            {
+                var properties = new List<Property>();
+                var meshPrimitives = MeshPrimitive.CreatePlaneFromTriangles();
+                meshPrimitives[0].Material = new Runtime.Material();
+                meshPrimitives[0].Material.MetallicRoughnessMaterial = new Runtime.PbrMetallicRoughness();
+                meshPrimitives[1].Material = new Runtime.Material();
+                meshPrimitives[1].Material.MetallicRoughnessMaterial = new Runtime.PbrMetallicRoughness();
 
-        public Runtime.GLTF SetModelAttributes(Runtime.GLTF wrapper, Runtime.Material material, List<Property> combo, ref glTFLoader.Schema.Gltf gltf)
-        {
-            foreach (var req in requiredProperty)
-            {
-                if (req.name == Propertyname.ExtensionUsed_SpecularGlossiness)
-                {
-                    // Initialize SpecGloss for every set
-                    material.Extensions = new List<Runtime.Extensions.Extension>();
-                    material.Extensions.Add(new Runtime.Extensions.PbrSpecularGlossiness());
-                    if (wrapper.ExtensionsUsed == null)
-                    {
-                        wrapper.ExtensionsUsed = new List<string>();
-                    }
-                    wrapper.ExtensionsUsed = wrapper.ExtensionsUsed.Union(
-                        new string[] { "KHR_materials_pbrSpecularGlossiness" }).ToList();
-                }
-                else if (req.name == Propertyname.BaseColorTexture)
+                // Apply the common properties to the gltf.
+                meshPrimitives[0].Material.MetallicRoughnessMaterial.BaseColorTexture = new Runtime.Texture { Source = baseColorTextureImage };
```

Comment:
create new Texture and set both to the same variable

Reply:
fixed